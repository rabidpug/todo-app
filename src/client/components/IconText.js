import { Icon, } from 'antd';
import React from 'react';

const IconText = ( {
  type, text, onClick,
} ) => (
  <span onClick={ onClick }>
    <Icon
      style={ { marginRight: 8, } }
      type={ type } />
    {text}
  </span>
);

export default IconText;
