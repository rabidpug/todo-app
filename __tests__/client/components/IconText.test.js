import IconText from 'Components/IconText';
import React from 'react';
import { shallow, } from 'enzyme';
import { shallowToJson, } from 'enzyme-to-json';

describe(
  'IconText', () => {
    it(
      'Should render the IconText', () => {
        const wrapper = shallow( <IconText /> );

        expect( shallowToJson( wrapper ) ).toMatchSnapshot();
      }
    );
  }
);
