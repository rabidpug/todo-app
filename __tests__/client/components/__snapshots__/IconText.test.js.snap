// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconText Should render the IconText 1`] = `
<span>
  <Icon
    style={
      Object {
        "marginRight": 8,
      }
    }
  />
</span>
`;
